command/forwarder/root.go

```diff
@@ -106,12 +106,19 @@ func Command() *cobra.Command {
 
 	templates.ActsAsRootCommand(cmd, nil, cg, FlagGroups(), EnvPrefix)
 
-	// Add other commands.
-	cmd.AddCommand(
-		grpctest.Command(), // hidden
-		httpbin.Command(),  // hidden
-		version.Command(),
+	// Add test commands.
+	test := &cobra.Command{
+		Use:   "test",
+		Short: "Run test servers for various protocols",
+	}
+	test.AddCommand(
+		grpc.Command(),
+		httpbin.Command(),
 	)
+	cmd.AddCommand(test)
```

Comment:
nit: Test command could be moved to `/command/test/test.go`

Reply:
It's not as grate as it sounds. The usefulness of such command is negligible.

Reply:
ok